tests/extendrtests/src/rust/Cargo.toml

```diff
@@ -19,7 +19,10 @@ extendr-api = "*"
 ## Build against local extendr sources. Check locally by 
 ## running rcmdcheck::rcmdcheck(check_dir = "../../../") from
 ## the root of extendrtests.
-extendr-api = { path = "../../../../../extendr/extendr-api/"}
+## The code is compiled in 
+## 'extendr/extendrtests_check/extendrtests.Rcheck/00_pkg_src/extendrtests/src/rust'
+## Cargo should go (relatively) 7 directories up (../) and then into './extendr/extendr-api'.
+extendr-api = { path = "../../../../../../../extendr/extendr-api/"}
```

Comment:
With this change, can you still build extendrtests locally from within RStudio by clicking on "Install and Restart"? That was the reason for the strange setup, that both the local build and the CI runs would work.

Reply:
Oh, sorry, I completely forgot about RStudio, I don't use it that much.
Let me try and see what happens.

Reply:
Yeah, it probably won't work.

As I said elsewhere, you can move everything one folder down and then you can put the check files into the root of extendr rather than parallel to the root. I don't have a strong opinion one way or another.

Reply:
E.g., `extendr/tests/R-integration/extendrtests/`

Reply:
@clauswilke,
I actually encountered another problem with paths -- something related to how `tar` stores paths and a limit of 100 symbols or so, so the smaller the (absolute) paths are the better. 
I'd like to first look for some non-intrusive solutions before moving everything around.

Reply:
@clauswilke,
With our relative path structure, neither "Test Package" nor "Check package" work from the RStudio menu, only "Install and Restart".

Reply:
Test package should work. It doesn't actually compile anything, it just runs R code. I just tried with current master and all tests passed.

Check package does not work, and I don't think there's a way to fix this. But it's also not that important for development work, what is important is that we can easily install the package and run the testing code.